python/cugraph-dgl/tests/test_cugraph_storage.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+try:
+    import cugraph_dgl
+except ImportError:
```

Comment:
For this and other places where this pattern is used in test files: I usually catch `ModuleNotFoundError`, since `ImportError` can also be raised if the importable is present but had another problem preventing it from being imported (missing symbol, SyntaxError, etc.), and catching `ImportError` will mask those errors we should probably know about.

Reply:
Changed to `ModuleNotFoundError` 
 https://github.com/rapidsai/cugraph/pull/2896/commits/aa394725f2ccab68257dfad8f8001c3e05d7a8b5